# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/test/gtest/child/TestFileSystemBackgroundRequestHandler.cpp
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/test/gtest/child/TestFileSystemBackgroundRequestHandler.cpp
--- a/dom/fs/test/gtest/child/TestFileSystemBackgroundRequestHandler.cpp	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/test/gtest/child/TestFileSystemBackgroundRequestHandler.cpp	Wed Aug 31 09:34:48 2022 +0000
@@ -9,31 +9,31 @@
 #include "gtest/gtest.h"
 #include "mozilla/SpinEventLoopUntil.h"
 #include "mozilla/UniquePtr.h"
-#include "mozilla/dom/OriginPrivateFileSystemChild.h"
-#include "mozilla/dom/POriginPrivateFileSystem.h"
+#include "mozilla/dom/FileSystemManagerChild.h"
+#include "mozilla/dom/PFileSystemManager.h"
 
 namespace mozilla::dom::fs::test {
 
 class TestFileSystemBackgroundRequestHandler : public ::testing::Test {
  protected:
   void SetUp() override {
-    mOPFSChild = MakeAndAddRef<TestOriginPrivateFileSystemChild>();
+    mFileSystemManagerChild = MakeAndAddRef<TestFileSystemManagerChild>();
   }
 
   UniquePtr<FileSystemBackgroundRequestHandler>
   GetFileSystemBackgroundRequestHandler() {
     return MakeUnique<FileSystemBackgroundRequestHandler>(
-        new TestFileSystemChildFactory(mOPFSChild));
+        new TestFileSystemChildFactory(mFileSystemManagerChild));
   }
 
   mozilla::ipc::PrincipalInfo mPrincipalInfo = GetPrincipalInfo();
-  RefPtr<TestOriginPrivateFileSystemChild> mOPFSChild;
+  RefPtr<TestFileSystemManagerChild> mFileSystemManagerChild;
 };
 
 TEST_F(TestFileSystemBackgroundRequestHandler,
        isCreateFileSystemManagerChildSuccessful) {
-  EXPECT_CALL(*mOPFSChild, Shutdown()).WillOnce([this]() {
-    mOPFSChild->OriginPrivateFileSystemChild::Shutdown();
+  EXPECT_CALL(*mFileSystemManagerChild, Shutdown()).WillOnce([this]() {
+    mFileSystemManagerChild->FileSystemManagerChild::Shutdown();
   });
 
   bool done = false;
@@ -41,9 +41,7 @@
   testable->CreateFileSystemManagerChild(mPrincipalInfo)
       ->Then(
           GetCurrentSerialEventTarget(), __func__,
-          [&done](const RefPtr<OriginPrivateFileSystemChild>& child) {
-            done = true;
-          },
+          [&done](const RefPtr<FileSystemManagerChild>& child) { done = true; },
           [&done](nsresult) { done = true; });
   // MozPromise should be rejected
   SpinEventLoopUntil("Promise is fulfilled or timeout"_ns,